pkg/alertmanager/statefulset.go

```diff
@@ -239,6 +239,7 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config) (*appsv1.S
 		amArgs = append(amArgs, "--cluster.listen-address=")
 	} else {
 		amArgs = append(amArgs, "--cluster.listen-address=[$(POD_IP)]:9094")
+		amArgs = append(amArgs, "--cluster.reconnect-timeout=5m")
```

Comment:
we still support Alertmanager < v0.15 on a best-effort basis so it'd be better if this is added around L375.

Reply:
@simonpasquier thank you for the review. I have updated the PR